R/commonTTest.R

```diff
@@ -525,54 +525,40 @@ summarySEwithin <- function(data=NULL, measurevar, betweenvars=NULL, withinvars=
   xLimits <- range(pretty(c(xMin, xMax)))
   xBreaks <- unique(x)
   yBreaks <- pretty(range(xDens))
-  xLabels <- unique(grp)
-
-  p <- ggplot2::ggplot()
+  xLabels <- as.character(unique(grp))
```

Comment:
`levels(grp)`?

Reply:
I tried that at first but `levels` may change the order:
```r
set.seed(123)
fact <- sample(gl(10, 5, labels = letters[1:10]))
as.character(unique(fact))
# [1] "g" "c" "a" "i" "h" "j" "e" "f" "b" "d"
levels(fact)
# [1] "a" "b" "c" "d" "e" "f" "g" "h" "i" "j"
```
and the groups are plotted in the order of `unique(grp)`.